core/src/main/resources/lib/layout/layout.jelly

```diff
@@ -203,37 +201,26 @@ ${h.initPageVariables(context)}
                     <j:set var="userName" value="${user.fullName}"/>
                   </j:otherwise>
                 </j:choose>
-                <span style="white-space:nowrap">
-                  <a href="${rootURL}/${user.url}" class="model-link inside inverse"><b>${userName}</b></a>
-                  <j:if test="${app.securityRealm.canLogOut()}">
-                    |
-                    <a href="${rootURL}/logout"><b>${%logout}</b></a>
-                  </j:if>
-                </span>
+                <a href="${rootURL}/${user.url}" class="model-link inside inverse">${userName}</a>
+                <j:if test="${app.securityRealm.canLogOut()}">
+                  <a href="${rootURL}/logout"><i class="fa fa-sign-out"></i></a>
+                </j:if>
               </j:when>
               <j:otherwise>
                 <st:include it="${app.securityRealm}" page="loginLink.jelly" />
               </j:otherwise>
             </j:choose>
           </j:if>
-        </div>
-        <div class="searchbox hidden-xs">
-          <!-- search box -->
-          <j:set var="searchURL" value="${h.searchURL}"/>
-          <form action="${searchURL}" method="get" style="position:relative;" class="no-json" name="search">
-            <!-- this div determines the minimum width -->
-            <div id="search-box-minWidth"/>
-            <!-- this div is used to calculate the width of the text box -->
-            <div id="search-box-sizer"/>
-            <div id="searchform">
-              <input name="q" placeholder="${%search}" id="search-box" class="has-default-text" value="${request.getParameter('q')}" />
-              <st:nbsp />
-              <a href="${%searchBox.url}"><l:icon class="icon-help icon-sm" /></a>
-              <div id="search-box-completion" />
-              <script>createSearchBox("${searchURL}");</script>
-            </div>
-          </form>
-        </div>
+          </li>
+          <li class="searchbox">
+            <j:set var="searchURL" value="${h.searchURL}"/>
+            <form action="${searchURL}" method="get" class="no-json" name="search">
+              <input name="q" placeholder="${%Search}" id="search-box" class="has-default-text" value="${request.getParameter('q')}" />
+              <!-- <div id="search-box-completion" /> -->
```

Comment:
This part is still in progress. I would like to make a proposal for removing this behavior.